runtime/service_integ_test.go

```diff
@@ -470,7 +479,7 @@ func testMultipleExecs(
 		require.NoError(t, err, "failed to stat root path of jailer")
 		_, err = os.Stat(filepath.Join("/sys/fs/cgroup/cpu", cgroupPath))
 		require.NoError(t, err, "failed to stat cgroup path of jailer")
-		assert.Equal(t, filepath.Join("/firecracker-containerd", vmIDStr), cgroupPath)
+		assert.Regexp(t, fmt.Sprintf(".?.*/%s", vmIDStr), cgroupPath)
```

Comment:
Would it be `fmt.Sprintf(".+/%s", vmIDStr)`?

Reply:
Ah yea, that simplifies it. Forgot about the + meaning one or more.